leapp/workflows/__init__.py

```diff
@@ -338,8 +338,9 @@ def run(self, context=None, until_phase=None, until_actor=None, skip_phases_unti
                                      config_model=config_model, skip_dialogs=skip_dialogs)
                     try:
                         instance.run()
-                    except BaseException:
+                    except BaseException as exc:
                         self._unhandled_exception = True
+                        messaging.report_stacktrace(message=exc.message, trace=exc.exception_info, actorname=actor.name)
```

Comment:
@fernflower I like it! However I am thinking whether we could even improve it from the point of adding nice error msg like: "The {name} leapp actor failed unexpectedly", putting the traceback in details. And being able to provide normal error output like in case `StopActorExecutionError`. I mean, instead of total hard crash of the whole framework and printing just raw output.

WDYT about that? Not saying to do this in this PR, just curious about your thoughts here. The code looks good to me. I want to try it later yet.

Reply:
It is exactly what will be reported :) The exc.message is holding the 'The {name} leapp actor failed unexpectedly' and exc.exception_info has all the traceback, as has been added by @prilr (many thanks btw!).
Here is the except from the report file:
```
----------------------------------------
Risk Factor: high
Title: Actor bacula_check unexpectedly terminated with exit code: 1
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/leapp/repository/actor_definition.py", line 74, in _do_run
    actor_instance.run(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/leapp/actors/__init__.py", line 289, in run
    self.process(*args)
  File "/etc/leapp/repos.d/system_upgrade/common/actors/baculacheck/actor.py", line 20, in process
    0 / 0
ZeroDivisionError: division by zero

Summary: 
Key: 640a2258f1032d584a2cad87a1a70a57448337c8
----------------------------------------
```